src/transformers/image_utils.py

```diff
@@ -149,6 +149,26 @@ def to_numpy_array(self, image, rescale=None, channel_first=True):
 
         return image
 
+    def expand_dims(self, image):
+        """
+        Expands 2-dimensional `image` to 3 dimensions.
+
+        Args:
+            image (`np.ndarray` or `torch.Tensor`):
```

Comment:
Seems like a Pillow image is also accepted

Reply:
Edited the function comments